src/PowerShellEditorServices/Workspace/ScriptFile.cs

```diff
@@ -564,6 +579,43 @@ public BufferRange GetRangeBetweenOffsets(int startOffset, int endOffset)
 
         #region Private Methods
 
+        private static string GetPathAsClientPath(string path)
+        {
+            const string fileUriPrefix = "file:///";
```

Comment:
This doesn't seem to be used -- or at least is repeated below

Reply:
Fixed.